clang/include/clang/Basic/AttrDocs.td

```diff
@@ -5408,7 +5408,9 @@ considered inline.
 Not all targets support this attribute. ELF target support depends on both the
 linker and runtime linker, and is available in at least lld 4.0 and later,
 binutils 2.20.1 and later, glibc v2.11.1 and later, and FreeBSD 9.1 and later.
-Non-ELF targets currently do not support this attribute.
+MachO targets support it, but with slightly different semantics: the resolver is
```

Comment:
In docs and comments, we prefer the canonical spelling: Mach-O

Reply:
TIL, thanks!